internal/speakerlist/speakerlist.go

```diff
@@ -56,7 +56,12 @@ func New(logger log.Logger, nodeName, bindAddr, bindPort, secret, namespace, lab
 		return &sl, nil
 	}
 
-	mconfig := memberlist.DefaultLANConfig()
+	var mconfig *memberlist.Config
```

Comment:
let's init to DefaultWanConfig and override if the parameter is there

mconfig := memberlist.DefaultLANConfig()
if xxx {
   mconfig = memberlist.DefaultWANConfig()
}